modules/atf_logger.lua

```diff
@@ -86,6 +87,9 @@ local function is_hmi_tract(tract, message)
   return false
 end
 
+--- Calculate binary data size
+-- @tparam string binaryData Binary data of message
+-- @treturn number Binary data size
 local function getBinaryDataSize(binaryData)
```

Comment:
@aderiabin do we need function that just overrides `#` operator?

Reply:
@LuxoftAKutsan This function also checks existence of parameter and return 0 in case of it absence.
Without this check `#` produces error.